content/docs/replication/_index.md

```diff
@@ -34,6 +34,8 @@ CSM for Replication provides the following capabilities:
 
 > _**NOTE**_: To add or delete volumes to the existing SYNC Replication Group in PowerStore, the user needs to pause and then resume the replication group. For more details, please refer to the troubleshooting section.
 
+> _**NOTE**_: To delete last volume of the existing SYNC Replication Group in PowerStore, the user needs to first unassign the replication policy from the existing SYNC Replication Group. For more details, please refer to the troubleshooting section.
```

Comment:
From CSM perspective, let's talk about PV and replication group. And for resources on PowerStore, you can mention volume and volume group.
Also, there is no replication policy. It is replication rule set on a protection policy. You can simply mention 'protection policy'.

How about 'To delete the last PV from a SYNC Replication Group in PowerStore, the user needs to first unassign the protection policy from the volume group on the PowerStore Manager UI. For more details, refer to the troubleshooting section.' ?

PV comment applies to above Note as well.